fixture-monkey-api/src/main/java/com/navercorp/fixturemonkey/api/generator/ArbitraryGeneratorLoggingContext.java

```diff
@@ -0,0 +1,14 @@
+package com.navercorp.fixturemonkey.api.generator;
+
+
+public final class ArbitraryGeneratorLoggingContext {
```

Comment:
Please add the annotation.
`@API(since = "1.0.28", status = Status.EXPERIMENTAL)`